pages/linux/resolvectl.md

```diff
@@ -0,0 +1,33 @@
+# resolvectl
+
+> Resolve domain names, IPV4 and IPv6 addresses, DNS resource records, and services.
+> Introspect and reconfigure the DNS resolver.
+> More information: <https://www.freedesktop.org/software/systemd/man/resolvectl.html>.
+
+- Show DNS settings:
+
+`resolvectl status`
+
+- Resolve the IPv4 and IPv6 addresses for one or more domains:
+
+`resolvectl query {{domain}}`
+
+- Retrieve the domain of {{IP}}:
+
+`resolvectl query {{ip}}`
+
+- Retrieve MX record of {{domain}}:
+
+`resolvectl  --legend=no -t MX query  {{domain}}`
+
+- Resolve an SRV record, for example _xmpp-server._tcp gmail.com:
+
+`resolvectl service _{{service}}._{{protocol}} {{name}}`
+
+- Retrieve the public key from an email address with an OPENPGP Record:
```

Comment:
```suggestion
- Retrieve the public key from an email address with a PGP Record:
```

Reply:
I made a typo here. The DNS record type is called OPENPGPKEY, probably the longest one available. What do you think about changing it to that?

Reply:
What about this?
```suggestion
- Retrieve the public key from an email address from an OPENPGPKEY DNS record:
```